treecraft/gui/debug_console.py

```
@@ -0,0 +1,226 @@
+import sys
```

Ruff (0.8.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)